partcad/src/partcad/lint/schema.py

```
@@ -7,14 +8,29 @@
+
 
 class SchemaLinting(Linting):
     def __init__(self, name: str, schema: dict) -> None:
         super().__init__(name)
         self.schema = schema
 
+    def get_hash(self, name: str, target: str) -> CacheHash:
+        # The schema is half of what produced a finding: a cached result for an
+        # unchanged file is only valid while the schema it was checked against
+        # is the same one.
+        hash = super().get_hash(name, target)
```

Ruff (0.16.1)

[error] Variable `hash` is shadowing a Python builtin

(A001)